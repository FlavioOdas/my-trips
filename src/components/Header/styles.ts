import styled from "styled-components";

export const Header = styled.header`
  display: flex;
  justify-content: center;
  width: 100%;
  margin-bottom: 7rem;
  text-align: center;
  font-size: 3rem;
`