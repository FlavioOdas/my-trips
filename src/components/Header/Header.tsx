import React from 'react'

import * as S from './styles';

const Header = () => {
  return (
    <S.Header>
      <h1>My trips</h1>
    </S.Header>
  )
};

export default Header;