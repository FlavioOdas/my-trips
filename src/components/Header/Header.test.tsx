import { render } from '@testing-library/react';
import React from 'react';

import Header from './Header';

describe("Header", () => {
  it("should render correclty", () => {
    const component = render(<Header />);

    expect(component.asFragment).toMatchSnapshot();
  });
}); 