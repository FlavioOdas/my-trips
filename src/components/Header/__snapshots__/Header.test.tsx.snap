// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correclty 1`] = `[Function]`;
